packunused.hs

```diff
@@ -271,18 +287,21 @@ main = do
 
         return files
 
-    compBuildOrder = map fst . allComponentsInBuildOrder
-
 componentNameAndModules :: Bool -> Component -> (String, String, [ModuleName])
 componentNameAndModules addMainMod c  = (n, n2, m)
   where
     m = nub $ sort $ m0 ++ PD.otherModules (componentBuildInfo c)
 
     (n, n2, m0) = case c of
         CLib   ci -> ("library", "", PD.exposedModules ci)
```

Comment:
I'm missing something here that extracts the unqualComponentName of sub-libs; see http://hackage.haskell.org/package/Cabal-2.2.0.1/docs/Distribution-Types-Library.html#v:libName

Reply:
It's not missing, it's moved to line 154

Reply:
No :-)